### PR TITLE
Format by VS Code

### PR DESCRIPTION
**Pull request by Jay-o-Way**
Headers, newlines, etc.
Note: I undid the changes to rule https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md014.md "Dollar signs used before commands without showing output"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,24 +291,25 @@ import "C"
...
+ buf := unsafe.Pointer(C.CString(content))
+ bufsize := C.size_t(len(content))
+ C.zip_entry_write(zip, buf, bufsize)
 
-	C.zip_entry_close(zip)
+ C.zip_entry_close(zip)
 
-	C.zip_close(zip)
+ C.zip_close(zip)
 }
```
Comment (kuba--) [inline README.md:308]: This formatting is incorrect. I'm not saying the original one is good, but at least for C looks much better.
I would suggest to focus in this PR on source code formatting (to be consistent).

**Comment by kuba--** [comment]
Personally, I like _smaller_ letters. Paragraphs which start with `#` look too big for me.

**Review by kuba-- — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jay-o-Way) [reply README.md]: Done that :)

**Comment by Jay-o-Way** [comment]
> Paragraphs which start with `#`

...are called a Header :) Sounds like a personal thing to me.

**Comment by Jay-o-Way** [comment]
Thank you

**Pull request merged**